docs/config-files/config.py

```diff
@@ -0,0 +1,25 @@
+# Reference:
+
+# brew install libyaml
+# sudo python -m easy_install pyyaml
+# http://net-informations.com/python/file/open.htm
+# https://www.basinc.com/manuals/EC_epsilon/Techniques/ChronoI/ca
+
+import yaml
+
+def main():
+    config_file = open("config.yml","r") # open a file
+
+    loaded = yaml.load(config_file)
+    print("Name of the file: ", config_file.name)
+    print("Opening mode: ", config_file.mode)
```

Comment:
This strikes me as too chatty....does the user really need to know this?  In general, less chatter is better than more chatter or important messages get lost.

Reply:
Got it. It was more for me debugging. I will make sure to have succinct codes for the final version.

Reply:
ah, gotcha.  debugging print statements are always ok, but should be cleaned before merging.....